app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt

```diff
@@ -273,7 +275,7 @@ fun PatchesSelectorScreen(
         }
     }
 
-
+    val lazyListState = rememberLazyListState()
```

Comment:
This is quite far away from the actual patch list column. Call it something like `patchLazyListState` instead